boxoffice/mailclient.py

```diff
@@ -58,6 +59,24 @@ def send_line_item_cancellation_mail(line_item_id, refund_amount, subject="Ticke
 
 
 @job('boxoffice')
+def send_order_refund_mail(order_id, refund_amount, note_to_user):
+    with app.test_request_context():
+        order = Order.query.get(order_id)
+        subject = __("{item_collection_title}: Refund for receipt no. {invoice_no}".format(
```

Comment:
There might be a case where the user gets a full refund (eg: ticket buyer who gets selected as a speaker), but whose ticket stays confirmed. The body of the email which explains the amount refunded should clarify this?